yt_dlp/extractor/bbc.py

```diff
@@ -317,16 +317,26 @@ def _raise_extractor_error(self, media_selection_error):
 
     def _download_media_selector(self, programme_id):
         last_exception = None
+        formats, subtitles = [], {}
         for media_set in self._MEDIA_SETS:
             try:
-                return self._download_media_selector_url(
+                fmts, sttls = self._download_media_selector_url(
```

Comment:
`sttls` is less obvious imo
```suggestion
                fmts, subs = self._download_media_selector_url(
```

Reply:
Agree.

(This was directly from the patch proposal as referenced in the commit message.)